ark/utils/segmentation_utils.py

```diff
@@ -215,10 +215,6 @@ def save_segmentation_labels(segmentation_dir, data_dir, output_dir,
         # generates segmentation borders and labels
         labels = labels.loc[fov, :, :, 'whole_cell'].values
 
-        # save the labels respectively
-        save_path_seg_labels = os.path.join(output_dir, f'{fov}_segmentation_labels.tiff')
-        io.imsave(save_path_seg_labels, labels, plugin='tifffile', check_contrast=False)
-
         # define borders of cells in mask
         contour_mask = find_boundaries(labels, connectivity=1, mode='inner').astype(np.uint8)
         contour_mask[contour_mask > 0] = 255
```

Comment:
`contour_mask` is being generated here but it doesn't look like it's used anywhere else in the function.

Reply:
@cliu72 yeah I was thinking about opening an issue for this one too, I think this function was actually supposed to save `contour_mask` to `save_path_seg_borders`, and not `labels`. @ngreenwald can you confirm?